pkg/engine/source/filesystem.go

```diff
@@ -436,6 +436,8 @@ func getPlatform(metadataPlatform string) string {
 		return "googleDeploymentManager"
 	case "Buildah":
 		return "buildah"
+	case "Pulumi":
```

Comment:
To avoid the go lint error, I suggest something like this:

```
 type suportedPlatforms map[string]string

 supportedPlatforms := &suportedPlatforms{
            "Ansible": "ansible",
            "CloudFormation": "cloudFormation",
            (...)
    }

 func getPlatform(platform string) string {
    if p, ok := supportedPlatforms[platform]; ok {
           return p
    }
    return "unknown"
 }
```

Please, note that this is only a suggested "approach"